Source/town.cpp

```diff
@@ -1142,15 +1142,15 @@ void T_DrawZoom(int x, int y)
 	if (chrflag || questlog) {
 		nSrcOff = SCREENXY(112, 159);
 		nDstOff = SCREENXY(320, 350);
-		wdt = 160;
+		wdt = (SCREEN_WIDTH - 320) / 2;
 	} else if (invflag || sbookflag) {
 		nSrcOff = SCREENXY(112, 159);
 		nDstOff = SCREENXY(0, 350);
-		wdt = 160;
+		wdt = (SCREEN_WIDTH - 320) / 2;

```

Comment:
`PANEL_WIDTH`/2 instead of `320`?

Reply:
Can't really decided if that would be the proper thing to do, or if we should define a value for the sbook/inv/qlog/char panel size.